docs/source/notes/numerical_accuracy.rst

```diff
@@ -51,6 +51,44 @@ datatype. E.g.:
     a.norm() # produces tensor(inf)
     a.double().norm() # produces tensor(1.4142e+20, dtype=torch.float64), representable in fp32
 
+.. _Linear Algebra Stability:
+
+Linear algebra (``torch.linalg``)
+---------------------------------
+
+Functions within ``torch.linalg`` are particularly problematic when dealing with extremal values.
```

Comment:
"Particularly problematic" is a vague phrase. In general, the handling of extremal values is not necessarily a (numerical) "stability" issue, either.